frontend/tailwind.config.js

```diff
@@ -138,19 +175,41 @@ module.exports = {
 		},
 		fontFamily: {
 			'primary': [
-				'sans-serif',
+				'neue-haas-unica',
+        '-apple-system',
+        'BlinkMacSystemFont',
```

Comment:
I think we should cleanup this file by removing some project-specific styles like font families, just keeping some standard fonts like Arial or something like that.

Reply:
I'll remove the project-specific fonts